src/Tribe/Views/V2/Views/Widgets/Widget_List_View.php

```diff
@@ -86,10 +86,11 @@ protected function setup_template_vars() {
 		$template_vars = parent::setup_template_vars();
 
 		// Here update, add and remove from the default template vars.
-		$template_vars['view_more_link']     = tribe_get_events_link();
-		$template_vars['widget_title']       = $this->context->get( 'widget_title' );
-		$template_vars['no_upcoming_events'] = $this->context->get( 'no_upcoming_events' );
-		$template_vars['show_latest_past']   = false;
+		$template_vars['events'] = [];
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 21 spaces but found 1 space

<sub>hash: ea5b4eac00ff4d35594cd826bced5afb</sub>